packages/components/src/slot-fill/index.js

```diff
@@ -44,13 +44,14 @@ export function Provider( { children, ...props } ) {
 	);
 }
 
-export function createSlotFill( name ) {
-	const FillComponent = ( props ) => <Fill name={ name } { ...props } />;
-	FillComponent.displayName = name + 'Fill';
+export function createSlotFill( key ) {
+	const baseName = typeof key === 'string' ? key : key.description;
```

Comment:
I think it might be good to flip this.

It would provide better back-compat in case anyone is doing anything weird and already using non-string keys/names (e.g. implementing `toString`):

```suggestion
	const baseName = typeof key === 'symbol' ? key.description : key;
```

Reply:
Good idea, I'll make that improvement. Thanks.

Reply:
✅ https://github.com/WordPress/gutenberg/commit/a40673fac1671cbc293b46ef30eb828a220f1e97